constructor/utils.py

```diff
@@ -130,14 +130,19 @@ def if_repl(match):
 
 
 def add_condarc(info):
+    from .conda_interface import MatchSpec  # prevent circular import
```

Comment:
Added the import here based on how it was imported in another function in this same file

```py
def parse_virtual_specs(info) -> dict:
    from .conda_interface import MatchSpec  # prevent circular import
```